common/scripts/configs/system_settings.sql.template

```diff
@@ -43,11 +43,16 @@ do $$
         "mktgPageAggsLastDtTm" timestamp with time zone,
         "mktgCTAAggsLastDtTm" timestamp with time zone,
         "defaultMinionInstanceSize" VARCHAR(255),
+        "defaultClusterType" VARCHAR(255) NOT NULL DEFAULT 'dynamic__x86_64__Ubuntu_14.04__c4.large',
         "createdAt" timestamp with time zone NOT NULL,
         "updatedAt" timestamp with time zone NOT NULL
       );
     end if;
 
+    if not exists (select 1 from information_schema.columns where table_name = 'systemSettings' and column_name = 'defaultClusterType') then
+      alter table "systemSettings" add column "defaultClusterType" VARCHAR(255) NOT NULL DEFAULT 'dynamic__x86_64__Ubuntu_14.04__c4.large';
```

Comment:
No foreign keys as this is the first table we create. The dropdown in UI will make sure that we only get the required values by default.